develop/parachain-devs/runtime-development/FRAME/add-pallet-to-the-runtime.md

```diff
@@ -142,4 +142,12 @@ Finally, to compose the runtime, update the list of pallets in the same file by
 ```
 
 !!! note
-    The `#[frame_support::runtime]` macro wraps the runtime's configuration, automatically generating boilerplate code for pallet inclusion.
\ No newline at end of file
+    The `#[frame_support::runtime]` macro wraps the runtime's configuration, automatically generating boilerplate code for pallet inclusion.
+
+## Next Steps
```

Comment:
I don't think these next steps make sense. Ideally, you test the pallet, benchmark it, and then add it to the runtime.

Reply:
If we're going to keep this type of section, I'd recommend changing the title to "Where to Go Next" because this  is used on other pages. It would be nice to bake this type of section into the standard structure of a doc